docs/ossm/create-mesh/README.md

```diff
@@ -0,0 +1,201 @@
+# Adding services to a service mesh
+OpenShift Service Mesh 3 is very different compared to OpenShift Service Mesh 2 in a way how one can add a service to your service mesh. OSSM 3 is much closer to the [Istio](https://istio.io/) project. Detailed list of OSSM 3 vs. OSSM 2 changes is available in [Compared to OpenShift Service Mesh 2](../ossm2-vs-ossm3.md) chapter.
```

Comment:
Let's not include reference to OpenShift Service Mesh 2 on this page, because this page will live on even after OSSM 2 is end of life, and many users will be starting with OSSM 3 without any knowledge of OSSM 2. We should only refer to "OpenShift Service Mesh" here, which can assumed to be OSSM 3. I would assume the reader is fairly new to service mesh, so you may have to explain some concepts...if anything, they may have some familiarity with community Istio.

Reply:
Removed ossm2 references and added a bit more info to introduction.